core/database/migrations/20160723182900_createSchema.sql

```diff
@@ -100,6 +100,14 @@ CREATE TABLE IF NOT EXISTS "forks_stat" (
   "cause"             INT         NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS "multisignatures" (
```

Comment:
I wonder if we should remove all functionality, corresponding to multisignatures?